src/compiler/diagnosticMessages.json

```diff
@@ -4284,6 +4284,14 @@
         "category": "Error",
         "code": 7052
     },
+    "Element implicitly has an 'any' type because expression of type '{0}' can't be used to index type '{1}'.": {
+        "category": "Error",
+        "code": 7053
+    },    
+    "No index with a parameter of type '{0}' was found on type '{1}'.": {

```

Comment:
Can you change this to "No index signature with a parameter of type '{0}' was found on type '{1}'."?